config.yaml

```diff
@@ -135,6 +135,12 @@ client:
     ## By default Agent assumes only alphanumeric characters as part of the SDK Key string.
     ## https://github.com/google/re2/wiki/Syntax
     sdkKeyRegex: "^\\w+(:\\w+)?$"
+    ## Disable odp
+    disableOdp: false
+    ## Max number of items that can be stored in odp segments cache.
+    segmentsCacheSize: 10000
+    ## Timeout in seconds after which an item stored in odp segments cache will be deleted.
+    segmentsCacheTimeout: 600s
```

Comment:
What about grouping all these and future odp options under "odp". I think "odpCache" is also can be a part of this group. We can also clarify "odpCache" will override these values.

Reply:
What about:
-  odp_segment_request_timeout
- odp_event_request_timeout
- odp_event_flush_interval

We use these in Python sdk ODP sdk settings, but don't see them exposed in go-sdk. Are we including them in agent config?

Reply:
Have only added those params which are directly available through the main factory constructor. Should we be adding these internal properties as well? @jaeopt

Reply:
@Mat001 request timeout's and flush interval can also be updated in go-sdk but for that we need to supply instances of both the event manager and segment manager to the odp manager.

Reply:
@Mat001 a good call! I think it's valuable to provide those options (segment_request_timeout, event_request_timeout, and probably event_flush_interval as well) to agent customers. @yasirfolio3